### PR TITLE
Feature dark mode design

### PR DESCRIPTION
**Pull request by laurent-b4y**
The dark theme, for when the option is available.

### CODE REVIEW

File: dist/orejime.css [hunk 1]
```diff
@@ -0,0 +1,349 @@
+/*! orejime v1.2.3 - BSD-3-Clause license, original work Copyright (c) 2018 DPKit, modified work Copyright (c) 2019 Empreinte Digitale, all rights reserved. */
```
Comment (j1mb0b) [inline dist/orejime.css]: remove dist css and replace with src scss, also will you not add a unique scss file for dark mode?

Comment (j1mb0b) [reply dist/orejime.css]: Maybe we should also rename orejime to orejime-light, so we end up with orejime-dark.scss and orejime-light.scss

Comment (laurent-b4y) [reply dist/orejime.css]: Yes might be better... I'll do that make 2 different scss files

File: src/scss/orejime-dark.scss [hunk 2]
```diff
@@ -2,13 +2,13 @@ $orejime-box-shadow: 0 4px 6px 0 rgba(0, 0, 0, 0.2), 5px 5px 10px 0 rgba(0, 0, 0
 $orejime-border-radius: 0 !default;
```
Comment (j1mb0b) [inline src/scss/orejime-dark.scss:2]: @laurent-b4y just need the orejime-light now, you can add it to this branch since its introducing the distinction.

**Pull request closed without merging**